.gitignore

```diff
@@ -119,3 +119,6 @@ reports/
 
 # End of https://www.gitignore.io/api/android,intellij
 /.idea/misc.xml
```

Comment:
ops sorry i forgot to remove this file, it's just a plugin that allows pushing apk's to slack, i have to remove it